robots/flakefinder/main.go

```diff
@@ -36,26 +36,30 @@ import (
 	"k8s.io/test-infra/prow/flagutil"
 )
 
-func flagOptions() options {
-	o := options{
+func flagOptions() Options {
+	o := Options{
 		endpoint: flagutil.NewStrings("https://api.github.com"),
 	}
 	flag.IntVar(&o.ceiling, "ceiling", 100, "Maximum number of issues to modify, 0 for infinite")
 	flag.DurationVar(&o.merged, "merged", 24*7*time.Hour, "Filter to issues merged in the time window")
 	flag.Var(&o.endpoint, "endpoint", "GitHub's API endpoint")
 	flag.StringVar(&o.token, "token", "", "Path to github token")
-	flag.BoolVar(&o.isPreview, "preview", false, "if report should be written to preview directory")
+	flag.BoolVar(&o.IsPreview, "preview", false, "Whether report should be written to preview directory")
+	flag.StringVar(&o.PRBaseBranch, "pr_base_branch", PRBaseBranchDefault, fmt.Sprintf("Base branch for the PRs (default: '%s')", PRBaseBranchDefault))
+	flag.StringVar(&o.ReportOutputChildPath, "report_output_child_path", "", fmt.Sprintf("Child path below the main reporting directory '%s' (i.e. 'master', default is '')", ReportsPath))
 	flag.Parse()
 	return o
 }
 
-type options struct {
-	ceiling         int
-	endpoint        flagutil.Strings
-	token           string
-	graphqlEndpoint string
-	merged          time.Duration
-	isPreview       bool
+type Options struct {
```

Comment:
Why did you need to export Options? It's all in the main package

Reply:
Intellij complied about it not being exported.

Reply:
![image](https://user-images.githubusercontent.com/809335/65592516-02b14480-df8f-11e9-823c-d35c737005be.png)